src/main/scala/mesosphere/marathon/tasks/MarathonTasks.scala

```diff
@@ -32,4 +32,30 @@ object MarathonTasks {
       .setSlaveId(slaveId)
       .build
   }
+
+  /**
+    * Returns the host address of the supplied task as a string.
+    *
+    * If the supplied task has at least one NetworkInfo with an IP address
+    * filled in, this function returns the first such address.
+    *
+    * In all other cases, this function returns the slave host address.
+    */
+  def hostAddress(task: MarathonTask): String = {
```

Comment:
I am not sure if I like the method name. "host address" has historically meant the IP of the Mesos Agent and not the container network address.